src/Validators/ProcessingResult.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace OpenEMR\Validators;
+
+/**
+ * OpenEMR Service Processing Result
+ * 
+ * Data contained within a processing result includes:
+ * - isValid: indicates if the data provided to the service was valid
+ * - validatiomMessages: validation errors, if any, which occurred during processing
+ * - processingErrors: system related errors, if any, which occured during processing
+ * - data: the return value of the operation/process
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Dixon Whitmire <dixon.whitmire@ibm.com>
+ * @copyright Copyright (c) 2020 Dixon Whitmire <dixon.whitmire@ibm.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+class ProcessingResult
```

Comment:
This class is used to encapsulate and convey results from validation and/or a "service" to its caller in a uniform manner.
This prevents the caller from having to know how [Particle Validation](http://validator.particle-php.com/en/latest/) packages it's results or works.